src/plugins/hetero/src/config.cpp

```diff
@@ -4,70 +4,53 @@
 
 #include "config.hpp"
 
-#include "ie/ie_plugin_config.hpp"
 #include "openvino/runtime/internal_properties.hpp"
 #include "openvino/runtime/properties.hpp"
 
 using namespace ov::hetero;
 
-Configuration::Configuration() : dump_graph(false) {}
+Configuration::Configuration() {}
 
 Configuration::Configuration(const ov::AnyMap& config, const Configuration& defaultCfg, bool throwOnUnsupported) {
-    OPENVINO_SUPPRESS_DEPRECATED_START
     *this = defaultCfg;
 
     for (const auto& it : config) {
         const auto& key = it.first;
         const auto& value = it.second;
 
-        if (HETERO_CONFIG_KEY(DUMP_GRAPH_DOT) == key) {
-            dump_graph = value.as<bool>();
-        } else if ("TARGET_FALLBACK" == key || ov::device::priorities == key) {
+        if ("TARGET_FALLBACK" == key || ov::device::priorities == key) {
```

Comment:
`TARGET_FALLBACK` is also an obsolete name. Users need to use `ov::device::priorities`